villainary-react/app/profile/loading.tsx

```diff
@@ -0,0 +1,5 @@
+import WavyLoader from '../components/WavyLoader';
+
+export default function loading() {
+    return <WavyLoader text="mwahaha" />;
+}
```

Comment:
Why is this code duplicated so many times?  Is it possible to make this more dry?

Reply:
the way loading works in next js, is for each route, you have a loading.tsx file in the directory, and it automatically picks it up.